Detectors/TRD/workflow/include/TRDWorkflow/T0FitSpec.h

```diff
@@ -0,0 +1,177 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef O2_TRD_T0FITSPEC_H
+#define O2_TRD_T0FITSPEC_H
+
+/// \file   T0FitSpec.h
+/// \brief  DPL device for steering the TRD t0 fits
+/// \author Luisa Bergmann
+
+#include "TRDCalibration/T0Fit.h"
+#include "DetectorsCalibration/Utils.h"
+#include "DataFormatsTRD/PHData.h"
+#include "DataFormatsTRD/CalT0.h"
+#include "CommonUtils/MemFileHelper.h"
+#include "Framework/Task.h"
+#include "Framework/ConfigParamRegistry.h"
+#include "Framework/ControlService.h"
+#include "Framework/WorkflowSpec.h"
+#include "CCDB/CcdbApi.h"
+#include "CCDB/CcdbObjectInfo.h"
+#include "Framework/CCDBParamSpec.h"
+#include "DetectorsBase/GRPGeomHelper.h"
+#include <chrono>
+
+#include "TH2.h"
+#include "TH1.h"
```

Comment:
I don't think the histogram includes are needed here and would remove them

Reply:
removed the histogram includes (here and in T0Fit.h)